### PR TITLE
[18.0][MIG] base_user_show_email: Migration to 18.0.

### PR DESCRIPTION
**Pull request by lef-adhoc**
I migrated it through this [PR](https://github.com/OCA/server-auth/pull/605)

### CODE REVIEW

**Review by JrAdhoc — APPROVED** [review]
LGTM functional and code review

**Review by lav-adhoc — APPROVED** [review]

**Review by matiasperalta1 — APPROVED** [review]
Functional and Technical review

File: base_user_show_email/views/res_users_view.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
         </field>
     </record>
     <record id="user_email_tree" model="ir.ui.view">
-        <field name="name">res.users.show_email.tree</field>
+        <field name="name">res.users.show_email.list</field>
```
Comment (dreispt) [inline base_user_show_email/views/res_users_view.xml:23]: Not blocking, but this is an unnecessary change that can break extensions inheriting it.

**Review by dreispt — APPROVED** [review]
